ompi/mca/io/romio321/romio/adio/common/ad_read_coll.c

```diff
@@ -811,32 +809,34 @@ static void ADIOI_R_Exchange_data(ADIO_File fd, void *buf, ADIOI_Flatlist_node
 
     if (buftype_is_contig) {
 	j = 0;
-	for (i=0; i < nprocs; i++) 
+	for (i=0; i < nprocs; i++) {
 	    if (recv_size[i]) {
 		MPI_Irecv(((char *) buf) + buf_idx[i], recv_size[i], 
 		  MPI_BYTE, i, myrank+i+100*iter, fd->comm, requests+j);
 		j++;
 		buf_idx[i] += recv_size[i];
 	    }
-    }
-    else {
+        }
+    } else {
 /* allocate memory for recv_buf and post receives */
 	recv_buf = (char **) ADIOI_Malloc(nprocs * sizeof(char*));
-	for (i=0; i < nprocs; i++) 
+	for (i=0; i < nprocs; i++) {
```

Comment:
I think that the only _required_ pair of `{}` that is missing is on this line and between old lines 839 and 840.  I don't think we want to outdent the `j=0; for (i=0; i < nprocs; i++)...` chunk of text -- I think that's all part of the same block that should be protected by the missing `{}`.